PurchasesTests/Purchasing/PurchaserInfoTests.swift

```diff
@@ -306,7 +307,7 @@ class BasicPurchaserInfoTests: XCTestCase {
         ] as [String : Any]
         let purchaserInfoWithoutRequestData = Purchases.PurchaserInfo(data: response)
 
-        let entitlements: [String : Purchases.EntitlementInfo] = purchaserInfoWithoutRequestData!.entitlements.active
+        let entitlements: [String : EntitlementInfo] = purchaserInfoWithoutRequestData!.entitlements.active
```

Comment:
I'm somewhat concerned about this bit - we have a bit of a conflict where both the framework and its main class are called `Purchases`, so folks had to disambiguate using `Purchases.`, which won't be needed anymore... and I'm not sure it would compile if you do add it? We should try to figure out what implications this'll have (if any) for devs